TACC/Core-CMS-Custom
# bug/WP-181: Entity codes not saving on edit action

## Overview
Addresses bug where edits on an entity in 'Edit Registration' action were not written to database despite success message.
…

## Related

- [WP-181](https://jira.tacc.utexas.edu/browse/WP-181)
…

## Changes
- On edit registration template, edited entity form element names to differentiate from records on the same listing page (was causing some fields to not show on form at all/not be found when writing to database, causing silent failure)
- Updated action for updating entities via above form to look for updated element names
…

## Testing

1. Go to http://localhost:8000/administration/list-registration-requests/
2. On any record, open the 'Edit Record' option in the 'Actions' column drop down.
3. Make some edits to any of the fields in the entity section (some records here will not have entities (growing pains with local db testing), skip those as not representative of real data) and submit changes, ensure you are directed to a success page.
4. Navigate back to listing page and either View or Edit the same record and ensure changes you made now appear.

## UI

…
